app/models/landing_page/block/statistics.rb

```diff
@@ -30,15 +29,15 @@ def rows
       rows
     end
 
+    def attachment
+      @attachment ||= landing_page.attachments.find { |att| att.title == data["attachment_csv"] }
```

Comment:
Using title feels a bit brittle. Should this account for typos? i.e errors in case or extra whitespace, so we allow "Data One" to be the same as "Data one" and "Data  One" etc.

I do think title is better than using an id as I'm not sure how easy it would be to find the id in the publishing tool to configure the block correctly.

Reply:
Yeah, title is the one thing you have control of in the Whitehall UI, so it's possible to match up. ID is harder to find. We could use filename, that might be preferable?

Reply:
Changed to filename